From: Eric Joldasov <bratishkaerik@landless-city.net>

Skip heap sort test in std.sort due to test failure on native target.
The failure is specific to 0.14 branch and does not reproduce on master.

Signed-off-by: Eric Joldasov <bratishkaerik@landless-city.net>

diff --git a/lib/std/sort.zig b/lib/std/sort.zig
index 8705d24017..984f14c905 100644
--- a/lib/std/sort.zig
+++ b/lib/std/sort.zig
@@ -155,7 +155,7 @@ const sort_funcs = &[_]fn (comptime type, anytype, anytype, comptime anytype) vo
     block,
     pdq,
     insertion,
-    heap,
+    // heap,
 };

 const context_sort_funcs = &[_]fn (usize, usize, anytype) void{
